packages/supermassive/src/bin/supermassive.ts

```diff
@@ -66,15 +66,11 @@ function getFiles(inputs: Array<string>) {
     .flat()
     .filter(Boolean);
 }
-function getContextPath(
-  outputDir: string,
-  inputPath: string,
-  contextImport: string | undefined,
-) {
+function getContextPath(outputDir: string, contextImport: string | undefined) {
   if (!contextImport) {
     return;
   }
-  const contextDir = path.join(path.dirname(inputPath), contextImport);
+  const contextDir = path.join(process.cwd(), contextImport);
```

Comment:
Just FYI for future changes, this is equivalent to `path.resolve(contextImport)`